app/webpage/format.css

```diff
@@ -0,0 +1,129 @@
+div {
+  position: relative;
+  background: none !important;
+  border: none !important;
+}
+
+.row{
+  margin-top: 0 !important;
+  margin-bottom: 0 !important;
+}
+
+/* plan-head-02, plan-tlbr-02 */
+li {
+  cursor: pointer;
+}
+
+/* lgin-user */
+.front{
+  background: #99ccff !important;
+}
+
+/* lgin-user, lgin-pswd, lgin-npwd */
+.user-verify{
+  position: absolute;
+  bottom: 20px;
+  left: 0px;
+}
+
+/* lgin-npwd */
+.create{
+  border: 1px solid black !important;
+  margin: 5px;
+}
+
+/* plan-head */
+#title-bar{
+  background: #b3ffb3 !important;
+  border: 2px solid grey !important;
+}
+
+/* plan-head-01 */
+#header{
+  font-size: 4rem;
+  padding: .50rem;
+}
+
+/* plan-head-02 */
+#option-div{
+  height: 6rem !important;
+  width: 6rem !important;
+  position: absolute !important;
+  bottom: 0 !important;
+  left: 0 !important;
+}
+
+button#option-btn{
+  background: url(https://cdn1.iconfinder.com/data/icons/seo-13/512/settings-128.png);
+  height: 3rem;
+  width: 3rem;
+  background-size: 100%;
+  position: absolute;
+  bottom: 1rem;
+  left: 2rem;
+}
+
+/* plan-head-02-03 */
+#option-help{
+  margin-top: 5rem;
+}
+
+/* plan-wksl, plan-tlbr */
+#body{
+  height: 80rem;
+}
+
+/* plan-wksl */
+#schedule{
+  display: flex;
+  padding: 1rem;
+  border: 2px solid grey !important;
+}
+
+.week{
+  height: 100%;
+  flex-grow: 1;
+  flex-basis: 5rem;
+  display: flex;
+  flex-direction: column;
+}
+
+.task{
+  font-size: 1.5rem;
+  border: 1px solid #bdbdbd;
+  flex-grow: 1;
+}
+
+.day{
+  font-weight: bold;
+  font-size: 2.5rem;
+  border: 2px solid #686868 !important;
+}
+
+.task{
+  border: 1px solid lightgrey !important;
+}
+
+/* plan-tlbr */
+#tool-bar{
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+  background: #b3f0ff !important;
+  border: 2px solid grey !important;
+}
+
+/* plan-tlbr-02 */
+.wkday-menu{
+  padding: 0;
+  background: #fafafa;
+}
+
+.wkday{
+  border: 1px solid #e9e9e9;
+}
+
+/* plan-tlbr-03 */
+#tool-buttons{
+  padding-top: 3rem;
+}
```

Comment:
I was finding that I wasn't able to change Bootstrap's defaults without it. I'm happy to go back and see if I can find a different way, I only resorted to that after finding nothing else for a good while.